### DISCUSSION TITLE
Add window size 1/4, 2/4, 1/4

### DISCUSSION THREAD

**Discussion by aa-mudassir** [rxhanson/Rectangle#1523] [Ideas]
Can window sizes be created for 1/4, 2/4, and 1/4?

Currently, I've used an ultrawide screen and split my window into 3 part
I want the window on the  left side to be 1/4 and window in the center is 2/4 and the right size to be 1/4

Thank you

**Comment by rxhanson** [comment]
All of these sizes/positions are available. Is there something additional that you're looking for?

**Reply by Lzyct** [reply]
![image](https://github.com/user-attachments/assets/9b2f0f11-66ad-4bf8-8243-4fec991de2c5)

Where is the option for 2/4? @rxhanson

**Reply by Aaron-Rumpler** [reply]
"Center Half", 3rd from the top in the left column.

**Reply by Lzyct** [reply]
Ah, I see
Thank you @Aaron-Rumpler
